server/boards/routes.ts

```diff
@@ -770,4 +771,66 @@ router.get(
   }
 );
 
+/**
+ * @openapi
+ * /boards/{board_id}:
+ *   delete:
+ *     summary: Delete board.
+ *     operationId: deleteBoard
+ *     description: Deletes the specified board.
+ *     tags:
+ *       - /boards/
+ *     security:
+ *       - firebase: []
+ *     parameters:
+ *       - name: board_id
+ *         in: path
+ *         description: The external id of the board to delete.
+ *         required: true
+ *         schema:
+ *           type: string
+ *           format: uuid
+ *         examples:
+ *           existing:
+ *             summary: An existing board
+ *             value: c6d3d10e-8e49-4d73-b28a-9d652b41beec
+ *     responses:
+ *       401:
+ *         description: User is not logged in.
+ *         $ref: "#/components/responses/default401"
+ *       403:
+ *         $ref: "#/components/responses/ensureBoardAccess403"
+ *       404:
+ *         $ref: "#/components/responses/default404"
+ *       500:
+ *         description: Internal Server Error
+ *         $ref: "#/components/responses/default500"
+ *       204:
+ *         description: Board deleted.
+ */
+router.delete(
+  "/:board_id",
+  ensureLoggedIn,
+  ensureBoardAccess,
+  async (req, res) => {
+    const { board_id: boardExternalId } = req.params;
+
+    let currentUserId: string = req.currentUser!.uid;
+    log(`Deleting board with id: ${boardExternalId}`);
+    const deleteSuccessful = await deleteBoard({
+      boardExternalId,
+      firebaseId: currentUserId,
```

Comment:
As noted in my earlier comment, you shouldn't need the `firebaseId` in the `deleteBoard` function. But you might want to record the user preforming the delete in the log.